### PR TITLE
Switch to "Kalix Runtime"

### PR DESCRIPTION
**Pull request by ennru**
References
- https://github.com/lightbend/kalix/issues/9895
- https://github.com/lightbend/kalix-runtime/pull/2173

Supersedes
- #522

### CODE REVIEW

File: sdk/bin/prepare.sh [hunk 1]
```diff
@@ -21,12 +21,12 @@ if [ -d "./proto" ] ; then
...
-     # proxy project source directory
-     echo "Using snapshot of proxy and sdk protocols from '$PROXY_SNAPSHOT_DIRECTORY'"
-     cp -rf $PROXY_SNAPSHOT_DIRECTORY/protocols/proxy/src/main/protobuf/* ./proto/
-     cp -rf $PROXY_SNAPSHOT_DIRECTORY/protocols/sdk/src/main/protobuf/* ./proto/
+  if [ -n "$RUNTIME_SNAPSHOT_DIRECTORY" ]; then
+     # Use local runtime and sdk sources, useful for development, point RUNTIME_SNAPSHOT_DIRECTORY to the local
+     # runtime project source directory
+     echo "Using snapshot of proxy and sdk protocols from '$RUNTIME_SNAPSHOT_DIRECTORY'"
+     cp -rf $RUNTIME_SNAPSHOT_DIRECTORY/protocols/proxy/src/main/protobuf/* ./proto/
+     cp -rf $RUNTIME_SNAPSHOT_DIRECTORY/protocols/sdk/src/main/protobuf/* ./proto/
```
Comment (efgpinto) [inline sdk/bin/prepare.sh:29]: Where do we set this ENV VAR `RUNTIME_SNAPSHOT_DIRECTORY` ? I could not find it.. jsut making sure we update that piece as well

**Review by efgpinto — APPROVED** [review]
LGTM with a question.

File: sdk/bin/prepare.sh (hunk 1 above)
Comment (ennru) [reply sdk/bin/prepare.sh:29]: My understanding is that this done manually for local development when you want to develop against an unreleased Runtime version with sources on your machine.

Comment (efgpinto) [reply sdk/bin/prepare.sh:29]: ah right, makes sense 👍

**Pull request merged**